test/presenters/organisations_presenter_test.rb

```diff
@@ -120,7 +125,8 @@
           link: {
             text: "The Charity Commission",
             path: "/government/organisations/charity-commission",
-            context: nil
+            context: nil,
+            description: nil
```

Comment:
Do we need a test where both context and description aren't nil?

Reply:
I was going to add a test, but I just realised I'm deleting/modifying all this code in the next PR anyway so it's probably ok to leave as is until then